### PR TITLE
ref: reorg and wrap `import` usage block

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Prevents horizontal scrolling in https://dvc.org/doc/command-reference/import#synopsis.

> Edited from GH so couldn't combine with https://github.com/iterative/dvc.org/pull/3934, sorry.

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Edited from GH so couldn't combine with #3934, sorry.

File: content/docs/command-reference/import.md [hunk 1]
```diff
@@ -8,9 +8,10 @@ Download a file or directory tracked by another DVC or Git repository into the
 ## Synopsis
 
 ```usage
-usage: dvc import [-h] [-q | -v] [-j <number>]
-                  [-o <path>] [--file <filename>]
-                  [--rev <commit>] [--no-exec | --no-download] [--desc <text>]
+usage: dvc import [-h] [-q | -v] [--file <filename>]
```
Comment (shcheklein) [inline content/docs/command-reference/import.md:11]: if we change order, should we also update in DVC itself?

**Review by shcheklein — APPROVED** [review]

File: content/docs/command-reference/import.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/import.md:11]: Yes, that would be ideal cc @dberenbaum

Comment (dberenbaum) [reply content/docs/command-reference/import.md:11]: @jorgeorpinel Is this minor cleanup or is there a larger goal? If it's minor cleanup, would you mind submitting the accompanying core change PR in these cases?

Comment (jorgeorpinel) [reply content/docs/command-reference/import.md:11]: Def. minor. Yes, sorry I know I should just didn't have time but also didn't want to leave it as it is now and it's quick to fix on the docs side. Unfortunately we didn't catch this during which ever PR added flags that made the usage block overflow. I'll add it to the best/std practices (maybe we can make a check list) for writing/updating cmd refs.

Comment (dberenbaum) [reply content/docs/command-reference/import.md:11]: Thanks @jorgeorpinel! Do you plan to open the core PR (no rush, just clarifying)? I think it makes sense to set the precedent that whoever contributes a fix owns propagating the changes to both core and docs repos.

Comment (jorgeorpinel) [reply content/docs/command-reference/import.md:11]: I'll take care of it 🙂

Comment (dberenbaum) [reply content/docs/command-reference/import.md:11]: Thanks @jorgeorpinel!

Comment (jorgeorpinel) [reply content/docs/command-reference/import.md:11]: p.s. I just noticed that we did catch this but post-merge. See https://github.com/iterative/dvc.org/pull/3773#discussion_r964450724. There's a follow-up to that too: https://github.com/iterative/dvc.org/pull/3930 -- reviewing now.

**Pull request merged**